recipes/parallelproj/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.5" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 411fe7552106db2e20487f2429907bfac641020469b0e7593134975345148cf5
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe
+  # is exactly the same for all platforms. It is okay if the dependencies are
+  # not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python
+  # for more details.
+  # noarch: python
+
+  number: {{ build_number }}
+  skip: True  # [osx or win]
+  # only works for CUDA>=11.0
+  skip: True  # [linux and cuda_compiler_version == "10.2"]
+
+  # If the installation is complex, or different between Unix and Windows,
+  # use separate bld.bat and build.sh files instead of this key. By default,
+  # the package will be built for the Python versions supported by conda-forge
+  # and for all major OSs. Add the line "skip: True  # [py<35]" (for example)
+  # to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit
+  # to Windows.
+
+outputs:
+  - name: parallelproj
+    build:
+      skip: True  # [cuda_compiler_version != "None"]
+    script: build_openmp.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake >=3.23
+        - ninja
+        - libgomp  # [linux]
+        - llvm-openmp  # [osx]
+      run:
```

Comment:
This should be `host:` as well

Reply:
Hm. The problem occurs when the parallelproj-gpu is build. And there the libs are in the requirements :/
So (without testing), I'd guess that adding it to the host req. for the parallelproj output would change that.
The parallelproj output get build fine.

Reply:
Hm. Hard to say for me. Is there a way to test that?
Could it be related to this line here in the [CMakeList.txt](https://github.com/gschramm/parallelproj/blob/9483ad5122d076465e1511975657e89f7bec4122/cuda/CMakeLists.txt#L26)?